CDCgov/prime-reportstream
# 6559 Cleanup and simplification of custom FHIR extension URLs

This PR does some minor cleanup of some FHIR extension URLs.  The idea here was to make the extension names consistent before this gets too far down the production line.

[PreTranslation_to_State_CO.txt](https://github.com/CDCgov/prime-reportstream/files/9676141/PreTranslation_to_State_CO.txt)

Test Steps:
1. Use the HCA samples provided and convert them from HL7 to HL7.
2. Compare the output and verify no data was lost after this change.

## Changes
- Renamed some extension URLs to not include the resource they are used in
- Fixed PID-34 not being extracted from the right FHIR location
- Fixed getId() not returning the raw value when there is no URL prefix

## Checklist

### Testing

### Process

## Linked Issues
- #6559

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
